packages/elements-core/src/components/TryIt/Parameters/parameter-utils.ts

```diff
@@ -116,3 +119,48 @@ export function mapSchemaPropertiesToParameters(
     ...(required?.includes(name) && { required: true }),
   }));
 }
+
+export function toParameterSpec(jsonTreeNode: RegularNode): ParameterSpec {
+  const isBoolean = jsonTreeNode.primaryType === 'boolean';
+  // TODO: Why does ParameterSpec assume JSONSchema7 and ignore 6 and 4?
+  const schema = !isBoolean ? (jsonTreeNode.fragment as JSONSchema7) : undefined;
+  const examples =
+    !isBoolean && jsonTreeNode.fragment.examples && jsonTreeNode.fragment.examples[0]
+      ? [{ key: 'example', value: jsonTreeNode.fragment.examples[0] }]
+      : undefined;
+
+  return {
+    name: last(jsonTreeNode.path) ?? '<<UNKNOWN>>',
```

Comment:
do we do this magic unknown elsewhere?

Reply:
I don't know if we do this elsewhere, but I consider this to mostly be paranoia... and appeasing the type system, since `last()` returns `T | undefined`.  If the actual JSON path array was empty, which shouldn't occur in practice, this wouldn't really be broken but it would be strangely labeled in the UI.  FWIW, there's a similar short-circuit case in the `isRequired()` function we wrote together.

I'll make this easier to see by pulling the value out to a named variable with a comment to explain the null coalesce, and see what you think.

(As an aside, is it possible to meaningfully express "a non-empty array" in TypeScript?  I couldn't figure out how.)

Reply:
Take a look at 934903fa and let me know what you think.

Reply:
its fine. definitely could use `throw` expressions

Reply:
I considered throwing, but we can still produce an otherwise reasonable `ParameterSpec` instance, we just don't know its name/path.  Without some rules around what is/isn't a valid `ParameterSpec` (oh, how I miss constructors full of preconditions) I'm probably veering away from Postel's law, but trying to produce _something_.